README.md

```diff
@@ -134,6 +134,15 @@ New tests should be written where it makes sense. With Flywheel, the PHPUnit tes
 
 At this moment in time, the tests are not run automatically on public CI services.
 
+## Cypress Tests
+To run the tests you will need to: 
+
+- load the included SQL database, found in `/tests/cypress/fixtures/test-database.sql`
```

Comment:
How do I load this SQL?

Reply:
I could add a comment saying open your SQL database editor of choice, but past that it would change depending on what you choose

Reply:
Is it a complete WP database?

Reply:
Just trying to give a sense of context to it.

Reply:
I think so yes

Reply:
I would put that in the comment. 

"Replace your existing WP database with this test snapshot"